pages/events.js

```diff
@@ -30,7 +29,47 @@ export default function Events({ events }) {
       event.date.cfpClose ? new Date(event.date.cfpClose) > new Date() : false
     ),
   };
+  const filters = [
+    {
+      title: "Show all",
+      description: "List all events with no filters",
+      key: "all",
+      icon: FaListUl,
+      current: true,
+      total: categorisedEvents.all.length,
+    },
+    {
+      title: "CFP open",
+      description: "You can submit a talk to this conference",
+      key: "cfpOpen",
+      icon: FaMicrophoneAlt,
+      current: false,
+      total: categorisedEvents.cfpOpen.length,
+    },
+    {
+      title: "In person",
+      description: "These are in person events",
+      key: "inPerson",
+      icon: MdOutlinePeople,
+      current: false,
+      total: categorisedEvents.inPerson.length,
+    },
+    {
+      title: "Virtual",
+      description: "Held virtually online event",
+      key: "virtual",
+      icon: MdOutlineOnlinePrediction,
+      current: false,
+      total: categorisedEvents.virtual.length,
+    },
+  ];
+
+  let displayTabs = filters.flatMap((tab) => {
```

Comment:
Can't the `filters` variable be used directly? is `displayTabs` different? If not you could directly use `filter` or rename it appropriate if you like.

Reply:
i tried using the filters directly but let me try again

Reply:
Oh it definitely works directly with `filters`, now I'm confused as why even was i doing the `flatMap` lol

Reply:
Haha.. no worries